apps/api_web/test/api_web/controllers/prediction_controller_test.exs

```diff
@@ -141,6 +141,38 @@ defmodule ApiWeb.PredictionControllerTest do
     end
   end
 
+  test "versions before 2019-02-12 include Alewife platformed stops", %{conn: conn} do
```

Comment:
I'm confused about which versions include or don't include which stops - could you describe that? I thought earlier versions would be the ones that _didn't_ have the new platforms

Reply:
None of the versions make changes to the actual data. All they do is treat queries for old stop IDs as more expansive.

| Version | Stops expanded |
| -- | -- |
| 2019-02-12 | none |
| 2018-07-23 | Park St, Kenmore, Government Center, Alewife, Braintree, Forest Hills, Oak Grove |
| 2018-05-07 (and previous) | 2018-07-23 stops, plus North Station, South Station, Back Bay |

Reply:
so if an old-API user queries for a list of stops or something, they'll still get the new stop IDs?

Reply:
Yep; we didn't make that change even for the CR platform change and it didn't seem to break anything for people.

Reply:
OK cool; that seems good then. We'll have to make a post at some point about the new Green Line stop IDs too.